s3-launch-transfers.sh

```diff
@@ -738,6 +738,7 @@ do
 		  "${checksum_header[@]}"
 		  "${automakebucket_opts[@]}"
 		  "${derive_header[@]}"
+          --expect100-timeout 20
```

Comment:
Lines before and after are using tabs for indentation while this one is using spaces, so the alignment not even in some settings.
```suggestion
		  --expect100-timeout 20
```